pyproject.toml

```
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["setuptools>=61.0.0", "wheel"]
+
+[project]
+name = "format-converters"
```

```suggestion
name = "ocrd-format-converters"
```

There is a project https://pypi.org/project/format-converter which could be confused with this.